testsuite/MDAnalysisTests/coordinates/test_xdr.py

```diff
@@ -542,7 +553,10 @@ class _GromacsWriterIssue117(object):
 
     @pytest.fixture()
     def universe(self):
-        return mda.Universe(PRMncdf, NCDF)
+        with pytest.warns(UserWarning,
+                          match="ATOMIC_NUMBER record not found"):
```

Comment:
I'm so confused, why are testing a NetCDF file warning here? Should we use pytest.filterwarning if it's not something we need to explicitly test?

Reply:
My thinking was that it filters the warning and alerts us if anything is changing. Given how poor our handling of warning in tests is generally, that seemed reasonable.

However, I also see that it can be confusing to test something not directly related to the code in question (here: xdr).

What's the appropriate `filterwarnings` to use here?

Reply:
> it filters the warning and alerts us if anything is changing

We should already be testing that warning in the netcdf tests (I'm pretty sure we do because I tried to bring testing to 100% on that file last I touched it). I'll leave it to your decision here (feel free to dismiss my block, everything else looks good).

> What's the appropriate filterwarnings to use here?
Adding the following to the tests calling the fixture I think should do the trick:
`@pytest.mark.filterwarnings("ignore: ATOMIC_NUMBER record not found")`

Reply:
Thanks. Changed.